src/routes/pipeline.rs

```diff
@@ -5,7 +5,7 @@
 
 use crate::db;
 use crate::models::pipeline::{NewPipeline, PipelineChangeset, PipelineData, PipelineQuery};
-use crate::routes::error_handling::{ ErrorBody, default_500 };
+use crate::routes::error_handling::{default_500, ErrorBody};
```

Comment:
I notice that often in these PRs there is some reformatting that happens.  It's not an issue - I'm just wondering if you know why it might be happening.  It's usually something like this reordering, or whitespace changes.

Reply:
I run `cargo fmt` to do code formatting, and I usually try to do it before every commit, but sometimes I forget, so I think stuff like this probably results from me committing without formatting and then remembering to format on a subsequent commit.

Reply:
OK, cool.  That's what I figured.  It might be worth trying to automate this in the future (VERY low priority).